Documentation/create_project.md

```diff
@@ -0,0 +1,35 @@
+MonoGame distributes templates to set up a new project in two ways:
+
+- For Visual Studio
+- For .NET Core CLI
```

Comment:
We should add VS for Mac instructions too. 
@harry-cpp?

Reply:
Yup.